pkgs/development/python-modules/userpath/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, click
+, distro
+, tox
+, pytest
+, coverage
+}:
+
+buildPythonPackage rec {
+  pname = "userpath";
+  version = "1.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256="0mfjmvx286z1dmnrc7bm65x8gj8qrmkcyagl0vf5ywfq0bm48591";
+  };
+
+  propagatedBuildInputs = [ click distro ];
+
+  checkInputs = [ tox pytest coverage ];
+
+  # We should skip tox dependencies installation to run tests but
+  # tox doesn't have such an option yet (https://github.com/tox-dev/tox/issues/410)
+  doCheck = false;
+
+  checkPhase = ''
+    tox
+  '';
```

Comment:
we don't need to run tox, nixpkgs already gives us per-interpreter tooling.

however, I looked at the test suite and couldn't make heads or tails of it.


Anyway:

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

Reply:
Done.